### PR TITLE
add peace-of-posters 0.4.0

### PR DESCRIPTION
**Pull request by jonaspleyer**
I am submitting
- [x] a new package
- [ ] an update for a package

Description:
This package/template allows users to create scientific posters with minimal additional syntax.
It also comes with extensive documentation.

This PR was previously closed due to inactivity: https://github.com/typst/packages/pull/379

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by elegaanz** [comment]
Hi, and thanks for opening a new PR. There were a few points that were discussed in the previous one that would still need to be resolved:
- first, it is still recommended to use kebab-case for public functions and their arguments. But this is only a recommendation to be more consistent with other packages, and you can keep snake_case if you prefer.
- the package name is too canonical/obvious for what it does. Could you please rename it to something else? Adding an adjective or some distinctive prefix/suffix is enough.

**Comment by jonaspleyer** [comment]
Thanks for the immediate feedback. I am not quite sure how to handle this situation.

On the one hand, my package allows for deep customization of boxes which are used to build up the poster.
This way, not only one type of appearance can be generated but many more. The documentation does not properly show this at this moment but the functionality is already there. This makes it hard for me to find a suitable "specification" of the package name.
And this also makes it hard for me to justify publishing this as a template. Rather, I would prefer to publish 2 templates which use this package in order to construct a specific layout.

Please let me know your feedback. I can also come up with some mock-examples to show what I am getting at if desired.

**Comment by reknih** [comment]
Hey Jonas,

The naming rule is less about avoiding collisions with the standard library (and thus anticipating it, we would not want to ask that of our package authors 😅) but about no package advertising itself as THE package for something (like scientific posters) with its name.

For example, we asked the packages [`zine`](https://github.com/typst/packages/pull/502), [`iban-formatter`](https://github.com/typst/packages/pull/517), and [`palettes`](https://github.com/typst/packages/pull/517) to rename. They are now called `zen-zine`, `ibanator`, and `splash` which allows other packages to serve their niche and compete with them and also is more memorable and fun (arguabely).

You can totally publish template packages that depend on this package and you are free to submit them in one or multiple PRs, as convenient.

So if this is the tools package it could have a name like `lighthouse`, `poster-corner`, or `industrial-glue` (feel free to come up with your own), with the template packages having names that riff on those or are completely separate and more related to their appearance or use case.

By the way, I enjoyed [your blog post](https://jonas.pleyer.org/blog/2023/10/17/how-i-create-posters-with-typst/) about creating posters with Typst! Can we quote it on our site?

**Comment by jonaspleyer** [comment]
Hi,
Thanks for the detailed answer. I will rework my PR as suggested into a package and (possible multiple) template(s).
Of course you are very welcomed to quote my blog post. I would be very glad if you did. If you do not plan to publish this immediately I would ask for you to wait until I have reworked this PR such that users can enjoy this package/template when also having read the blog-post. I am back from my vacation now and will start working on it in the coming week.
Best,
Jonas

**Comment by jonaspleyer** [comment]
Hi Everyone,
I have changed the name to `peace-of-posters` and released it as a package instead of as a template. Furthermore, I have changed all naming conventions to use kebap-case instead of snake_case. I have made one very minor exception which is functions that are meant for private use which start (similarly to python naming conventions) with an underscore.
Please let me know what you think.

**Comment by reknih** [comment]
Name works, likewise variable naming conventions sound good

**Comment by jonaspleyer** [comment]
Thanks for the very fast response. I hope that this PR can be merged now.

File: packages/preview/peace-of-posters/0.4.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "peace-of-posters"
+version = "0.4.0"
+authors = ["Jonas Pleyer<jonas.sci@pleyer.org>"]
+license = "MIT"
+description = "Create scientific posters in Typst."
+entrypoint = "lib.typ"
+repository = "https://github.com/jonaspleyer/peace-of-posters"
+homepage = "https://jonaspleyer.github.io/peace-of-poster/"
```
Comment (elegaanz) [inline packages/preview/peace-of-posters/0.4.0/typst.toml:9]: This URL is missing a `s` in `posters`.
```suggestion
homepage = "https://jonaspleyer.github.io/peace-of-posters/"
```

**Review by elegaanz — COMMENTED** [review]
Just a typo in the homepage URL, after that it should be good to merge.

File: packages/preview/peace-of-posters/0.4.0/typst.toml (hunk 1 above)
Comment (jonaspleyer) [reply packages/preview/peace-of-posters/0.4.0/typst.toml:9]: Thanks for spotting that!

**Comment by reknih** [comment]
Thanks!

**Pull request merged**